Gemfile.jruby-1.9.lock

```diff
@@ -167,16 +167,16 @@ GEM
       i18n (= 0.6.9)
       jrjackson (~> 0.3.7)
       jruby-openssl (= 0.9.13)
-      logstash-core-event (= 2.3.1.snapshot1)
+      logstash-core-event (= 2.3.1)
       minitar (~> 0.5.4)
       pry (~> 0.10.1)
       rubyzip (~> 1.1.7)
       stud (~> 0.0.19)
       thread_safe (~> 0.3.5)
       treetop (< 1.5.0)
-    logstash-core-event (2.3.1.snapshot1-java)
-    logstash-core-plugin-api (1.7.0-java)
-      logstash-core (>= 2.0.0, <= 2.3.1.snapshot1)
+    logstash-core-event (2.3.1-java)
+    logstash-core-plugin-api (1.8.0-java)
```

Comment:
ok, thanks for the clarification

then go, go :shipit: when jenkins is green :LGTM: